src/hotspot/os/linux/cgroupV1Subsystem_linux.cpp

```diff
@@ -286,7 +286,7 @@ int CgroupV1Subsystem::cpu_shares() {
 
 char* CgroupV1Subsystem::pids_max_val() {
   GET_CONTAINER_INFO_CPTR(cptr, _pids, "/pids.max",
-                     "Maximum number of tasks is: %s", "%s %*d", pidsmax, 1024);
+                     "Maximum number of tasks is: %s", "%*d", pidsmax, 1024);
```

Comment:
According to https://docs.kernel.org/admin-guide/cgroup-v2.html#pid-interface-files, file `pid.max` either contains a number or the string `max`. So I think this should be (as we need to account for something not a number):

`"Maximum number of tasks is: %s", "%1023s", pidsmax, 1024);`

The conversion from `max` to unlimited happens here:
https://github.com/openjdk/jdk/blob/master/src/hotspot/os/linux/cgroupSubsystem_linux.cpp#L559